### PR TITLE
🚨 Testing New Existing Page

### PR DESCRIPTION
**Pull request by AlexanderEDavis**
## Description

Creation of newexisting page with new design for existing requests page. Checking to see if POC works and system will not break. Once testing is complete. system will be implemented into the main app and linked.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I read & comply with the [contributing guidelines](https://github.com/luketainton/FHeD/blob/main/.github/CONTRIBUTING.md)
- [x] I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [x] I have made corresponding changes the documentation (README.md).

### CODE REVIEW

File: .github/issue-branch.yml [hunk 1]
```diff
@@ -10,4 +10,6 @@ branches:
     prefix: epic/
   - label: type/docs
     prefix: docs/
+  - label: type/automation
+    prefix: automation/
```
Comment (luketainton) [inline .github/issue-branch.yml:14]: Pls change to auto/ - less characters to type when I'm switching branches :)

File: app/includes/header.php [hunk 2]
```diff
@@ -16,6 +16,9 @@
     <link rel="stylesheet" href="/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://cdn.materialdesignicons.com/5.3.45/css/materialdesignicons.min.css">
     <link rel="stylesheet" href="/css/custom.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
```
Comment (luketainton) [inline app/includes/header.php:21]: What's going on here?

File: .github/issue-branch.yml (hunk 1 above)
Comment (AlexanderEDavis) [reply .github/issue-branch.yml:14]: which line 13 or 14?

File: app/includes/header.php (hunk 2 above)
Comment (AlexanderEDavis) [reply app/includes/header.php:21]: Scripting required for bootstrap responsive tabs and dynamic content

File: .github/issue-branch.yml (hunk 1 above)
Comment (luketainton) [reply .github/issue-branch.yml:14]: 14

File: app/includes/header.php (hunk 2 above)
Comment (luketainton) [reply app/includes/header.php:21]: The CSS file is already hosted with the site (see above lines) and the JS is included in includes/footer

**Pull request merged**